src/sys/wait.rs

```diff
@@ -112,149 +96,52 @@ impl WaitStatus {
     }
 }
 
-#[cfg(any(target_os = "linux",
-          target_os = "android"))]
 mod status {
     use sys::signal::Signal;
-    use libc::c_int;
-    use libc::SIGTRAP;
+    use libc::{self, c_int, SIGTRAP};
 
     pub fn exited(status: i32) -> bool {
-        (status & 0x7F) == 0
+        unsafe { libc::WIFEXITED(status) }
```

Comment:
Everything in libc is automatically extern and unsafe, matching the underlying implementation if it didn't use macros.